modules/openshift-developer-cli-deleting-an-application.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.adoc
+// * cli_reference/openshift_developer_cli/creating-an-application-with-a-database.adoc
+// * cli_reference/openshift_developer_cli/creating-a-multicomponent-application-with-odo.adoc
+
+[id="deleting-an-application_{context}"]
+= Deleting an Application
+
+[IMPORTANT]
+====
+Deleting the application will delete all components associated with an application.
```

Comment:
Minor nitpick, but I would swap the "the" and "an" before the two application instances:

```
Deleting an application will delete all components associated with the application.
```